tests/bootstrap.php

```diff
@@ -65,22 +64,19 @@ static function () use ( $plugin_test_path, $plugin_name ) {
 		},
 		1
 	);
-} else {
-	// Force plugin to be active.
-	$GLOBALS['wp_tests_options'] = array(
-		'active_plugins' => array( basename( TESTS_PLUGIN_DIR ) . '/load.php' ),
-	);
 
-	// Add filter to ensure the plugin's admin integration is loaded for tests.
-	tests_add_filter(
-		'plugins_loaded',
-		static function () {
-			require_once TESTS_PLUGIN_DIR . '/includes/admin/load.php';
-			require_once TESTS_PLUGIN_DIR . '/includes/admin/server-timing.php';
-			require_once TESTS_PLUGIN_DIR . '/includes/admin/plugins.php';
-		},
-		1
-	);
+	if ( 'performance-lab' === $plugin_name ) {
```

Comment:
Doesn't the `--testsuite` need to always be supplied now? If `$plugi_bane` is empty, shouldn't an error occur?

Reply:
I have set the default test suite to `performance-lab`. See https://github.com/WordPress/performance/pull/1182/commits/7a5dea4b49c09b82bb12158901e12ba39f68960e